auction_keeper/strategy.py

```diff
@@ -104,7 +104,7 @@ def bid(self, id: int, price: Wad) -> Tuple[Optional[Wad], Optional[Transact], O
                 return None, None, None
 
             our_bid = Rad.min(Rad(bid.lot) * price, bid.tab)
-            our_price = price if our_bid < bid.tab else bid.bid / Rad(bid.lot)
+            our_price = price if our_bid < bid.tab else Wad(bid.bid) / bid.lot
```

Comment:
This was the cause of the bug first seen a few weeks back.  The `price` component of the tuple should always be expressed as a `Wad`, else there's a corner case where the transition from `tend` to `dent` could make it attempt to compare a `Rad` with a `Wad`, causing an exception.